code/perception/src/radar_node.py

```
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import rospy
+import ros_numpy
+import numpy as np
+import lidar_filter_utility
```

Ruff (0.7.0)

`lidar_filter_utility` imported but unused

Remove unused import: `lidar_filter_utility`

(F401)